### PR TITLE
New CPBL Scores App 

### PR DESCRIPTION
**Pull request by yuping917**
New CPBL app to display CPBL (Taiwan professional baseball league) game scores and schedules

# Description
Inspired by mlbscores, this is the first CPBL app to display [CPBL (Taiwan professional baseball league)](https://en.wikipedia.org/wiki/Chinese_Professional_Baseball_League) game scores and schedules for Tidbyt device. 

No API require for display past 5 games results for specify team (select from option), others does require [theSportDB](https://www.thesportsdb.com/) API key.

# Copilot

copilot:all

### CODE REVIEW

**Comment by yuping917** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by yuping917** [comment]
recheck

File: apps/cpblscores/cpbl_scores.star [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+            schema.Text(
+                id = "apikey",
+                name = "API Key",
+                desc = "Enter your theSportDB API key",
+                icon = "key",
+            ),
+        ],
+    )
+
+def get_cachable_data(url, ttl_seconds = CACHE_TTL_SECONDS):
```
Comment (matslina) [inline apps/cpblscores/cpbl_scores.star:440]: The http module now has built-in cache functionality. Just pass `tll_seconds` to `http.get()`. Would be great if you could switch over to using that!

Comment (yuping917) [reply apps/cpblscores/cpbl_scores.star]: Hey @matslina  - Thanks for the advice. I committed to adopting built-in cache functionality in http module.

**Pull request merged**